group2_assignment2_solution.ipynb

```diff
@@ -273,7 +271,7 @@
     "                                             'lang':'en',\n",
     "                                             'result_type':'recent', #INSERT YOUR CODE\n",
     "                                             'max_id':max_id_str, #INSERT YOUR CODE\n",
-    "                                             'geocode':'{},{},{}mi'.format(AUS_LAT, AUS_LONG,AUS_RADIUS) #INSERT YOUR CODE\n",
+    "                                             'geocode':'{},{},{}mi'.format(AUS_LAT,AUS_LONG,AUS_RADIUS) #INSERT YOUR CODE\n",
```

Comment:
If anything could we add spaces to this?

Reply:
There ya go 👌